src/Illuminate/View/Component.php

```diff
@@ -292,4 +325,121 @@ public function shouldRender()
     {
         return true;
     }
+
+    /**
+     * Flush the components cache.
+     *
+     * @return void
+     */
+    public static function flushCache()
+    {
+        static::$bladeViewCache = [];
+        static::$constructorParametersCache = [];
+        static::$methodCache = [];
+        static::$propertyCache = [];
+    }
+
+    /**
+     * Get the evaluated view contents for the given view.
+     *
+     * @param  string|null  $view
+     * @param  \Illuminate\Contracts\Support\Arrayable|array  $data
+     * @param  array  $mergeData
+     * @return \Illuminate\Contracts\View\View
+     */
+    public function view($view, $data = [], $mergeData = [])
```

Comment:
Note here: seems that, declaring this method, conflicts with the https://filamentphp.com's component class. So, shipping this new method, makes projects using older versions of https://filamentphp.com to break.

So, the filament team will have to adjust to this change, or we can ship this only in Laravel X.

Reply:
This new method is not required to have speed improvements, it's like a 7% speed improvement at scale.

Reply:
If this is merged we will merge filamentphp/filament#4465 to mitigate the problem